.kokoro

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands to stderr.
+set -x
+
+if [ ! -d .kokoro-ios-runner ]; then
+  git clone https://github.com/material-foundation/kokoro-ios-runner.git .kokoro-ios-runner
+fi
+
+pushd .kokoro-ios-runner
+git fetch > /dev/null
+TAG=$(git tag -l "v3*" | sort | tail -n1)
```

Comment:
Nit: Should the "v3" be a variable at the top of the script?  Is this the kokoro runner tag we want to check out?